scripts/core/queue.yml

```diff
@@ -332,3 +332,49 @@ params:
 returns:
     - $X_RESULT_ERROR_INVALID_QUEUE
     - $X_RESULT_ERROR_OUT_OF_HOST_MEMORY
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Get the suggested local work-item number from runtime implementation."
+class: $xQueue
+name: GetSuggestedLocalWorkSize
+ordinal: "0"
+details:
+    - "pLocalWorkSize can be omitted in urEnqueueKernelLaunch(), but beside from"
+    - "OpenCL, LocalWorkSize will need to be calculated or guessed before enqueue"
+    - "the kernel. This function will get the LocalWorkSize value used when enqueueing"
+    - "the kernel."
+    - "TODO: find a better place for this function"
```

Comment:
I feel like this should live in `scripts/core/kernel.yml`

Reply:
sure, let me move it to kernel.yml.